src/tools/wpa/ETWTraceEvent.cs

```diff
@@ -9,7 +9,11 @@
 
 namespace QuicEventDataSource
 {
+#pragma warning disable CA1036 // Override methods on comparable types
```

Comment:
Can you see if we can just remove the compare instead?

Reply:
Its not that, its because its a struct. FxCop basically requires all structs to implement comparable. But this doesn't make sense for this one as much.

Reply:
Can we ignore these globally instead?

Reply:
Yeah these I can globally disable.